packages/blockchain-link/jest.config.integration.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
         },
     },
     moduleFileExtensions: ['js', 'ts', 'json'],
-    testMatch: ['**/tests/integration/*.ts'],
+    testMatch: ['**/tests/integration/electrum.ts'],
```

Comment:
I set this to run only `electrum` since the rest of the tests in directory `packages/blockchain-link/tests/integration` are failing.  At least until we decide to fix them or to remove them.

Should we remove them?

Reply:
They are working, you need to call `yarn build:libs` and `yarn build:workers` before

Reply:
You are absolutely right! :+1:  I was confused, thanks! I updated that c725b5d81cb7e4961a76afe38f9e570ef8d0047d